src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -22,12 +22,27 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.37.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition="'$(TargetFramework)' == 'net9.0'">
+      <ItemGroup>
```

Comment:
You can simplify this by moving the condition to your ItemGroup (see line 46 below)

Reply:
Moved condition to ItemGroup element instead of Choose-When pattern